docs/references/backend/organization/get-organization-list.mdx

```diff
@@ -18,6 +18,7 @@ const organizations = await clerkClient.organizations.getOrganizationList();
 | `limit?` | `number` | The number of results to return. Must be an integer greater than zero and less than 501. |
 | `offset?` | `number` | The number of results to skip. |
 | `query?` | `string` | A search query to filter organizations by. |
+| `orderBy?` | `string` | The field to order by. Can be  `"name"`, `"updated_at"`, or `"members_count"`. Prefix with a `-` to reverse the order. Prefix with a `+` to list in ascending order. |
```

Comment:
```suggestion
| `orderBy?` | `"name" \| "updated_at" \| "members_count"` | The field to order by. Prefix with a `-` to reverse the order. Prefix with a `+` to list in ascending order.  |
```

Reply:
Hey @alexisintech! Your suggested change is the same as what I wrote initially, but @S3Prototype rewrote it to the current version in [`eedb7da` (#891)](https://github.com/clerk/clerk-docs/pull/891/commits/eedb7da957b8e481433f9dddb4cff8370b352cbe). Let me know what you guys prefer.

Reply:
I'm so sorry for the confusion. You were correct the first time by supplying the type of the prop. The type of `orderBy` is not simply a string, it is a defined union type https://github.com/clerk/javascript/pull/3164/files#diff-0023aebda073feedd66cfd55bc6a77ad38275c83a00d47fdbae558ef1d4e912bR25

so let's supply the correct type! you can go with my suggestion/your original solution

Reply:
Done :)